MediaStreamTrack.js

```diff
@@ -83,6 +83,16 @@ export default class MediaStreamTrack extends EventTarget(MEDIA_STREAM_TRACK_EVE
     this.muted = !this._enabled;
   }
 
+  _switchCamera() {
```

Comment:
You're absolutely right.  I'll also add a note to the readme, since it's hidden in the source otherwise.